packages/prisma-client/prisma/migrations/20220909124542_move_to_builds/migration.ts

```diff
@@ -0,0 +1,44 @@
+import { PrismaClient, Prisma } from "./client";
+import { v4 as uuidv4 } from "uuid";
+
+export default () => {
+  const client = new PrismaClient();
+  return client.$transaction(async (prisma) => {
+    const trees = await prisma.tree.findMany();
+    const projects = await prisma.project.findMany();
+
+    const builds: Prisma.BuildCreateManyInput[] = [];
+
+    for (const tree of trees) {
+      const project = projects.find(
+        (project) =>
+          project.devTreeId === tree.id ||
+          project.prodTreeId === tree.id ||
+          JSON.parse(project.prodTreeIdHistory).includes(tree.id)
+      );
+
+      if (project === undefined) {
+        continue;
+      }
+
+      builds.push({
+        pages: JSON.stringify({
+          homePage: {
+            id: uuidv4(),
```

Comment:
For `Page.id` I think it's better to use UUID instead of `bson-objectid` that we use for `Instance.id`. Main reason is that UUID can be easily created in an SQL query if we ever need to create a Page using just SQL.